usrsctplib/netinet/sctp_auth.c

```diff
@@ -1730,13 +1730,15 @@ sctp_handle_auth(struct sctp_tcb *stcb, struct sctp_auth_chunk *auth,
 	(void)sctp_compute_hmac_m(hmac_id, stcb->asoc.authinfo.recv_key,
 	    m, offset, computed_digest);
 
+#ifndef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
I would like to see a change only in the userland stack. So Maybe something like
```
#if defined(__Userspace__)
#ifndef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
if (timingsafe_bcmp(digest, computed_digest, digestlen) != 0) {
#else
if (0) {
#endif
#else
if (timingsafe_bcmp(digest, computed_digest, digestlen) != 0) {
#endif
```